slirp4netns.1.md

```diff
@@ -62,6 +62,11 @@ specify network namespace type ([path|pid], default=pid)
 **--userns-path=PATH** (since v0.4.0)
 specify user namespace path
 
+**--create-sandbox** (since v0.4.0)
+when running as a root (either on the host, or in a user namespace), create
```

Comment:
would it be possible to enter the target userns when non-root?

Reply:
added a patch for doing it